sonar-orchestrator/src/main/java/com/sonar/orchestrator/container/Server.java

```diff
@@ -275,48 +219,49 @@ Server restoreProfiles(ContentBody backup) {
    * @since 2.17
    */
   public void provisionProject(String projectKey, String projectName) {
-    adminWsClient().post("/api/projects/create",
-      "key", projectKey,
-      "name", projectName);
+    newHttpCall("/api/projects/create")
+      .setParam("key", projectKey)
+      .setParam("name", projectName)
+      .setAdminCredentials()
+      .execute();
+  }
+
+  /**
+   * @deprecated in 3.15. Replaced by {@link #newHttpCall(String)}.
+   */
+  @Deprecated
+  public String post(String relativePath, Map<String, Object> params) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)